### PR TITLE
fix(inbound-api): fixes for extension list and tls config

### PR DESCRIPTION
**Pull request by oderayi**

### CODE REVIEW

File: src/InboundServer/middlewares.js [hunk 1]
```diff
@@ -331,11 +331,12 @@ const createJwsValidator = (logger, keys, exclusions) => {
  * @return {Function}
  */
 const applyState = (sharedState) => async (ctx, next) => {
+    const outboundRequests = (sharedState.conf && sharedState.conf.mutualTLS && sharedState.conf.mutualTLS.outboundRequests) || {};
     Object.assign(ctx.state, {
         ...sharedState,
         conf: {
             ...sharedState.conf,
-            tls: sharedState?.conf?.mutualTLS?.outboundRequests,
+            tls: outboundRequests,
```
Comment (bushjames) [inline src/InboundServer/middlewares.js:339]: why this change?

**Review by bushjames — CHANGES_REQUESTED** [review]

File: src/InboundServer/middlewares.js (hunk 1 above)
Comment (oderayi) [reply src/InboundServer/middlewares.js:339]: @bushjames The previous construct was throwing error as it appears not be valid JavaScript.

**Review by bushjames — APPROVED** [review]
I cannot spot any issues.
